RELEASE/scripts/autoscend/auto_tower.ash

```diff
@@ -60,9 +60,13 @@ boolean L13_towerNSContests()
 					use_familiar($familiar[Oily Woim]);
 				}
 
-				provideInitiative(400, true);

```

Comment:
It looks like it was replaced with a 2 step process
1. switch familiar to oily woim if have it. (which would work better using the autoUseFamiliar() function I made that is waiting on the other pull)
2. autoMaximize("initiative -equip snow suit", 1500, 0, false);

Since nowadays it is preferred to do maximizations through autoMaximize.
What I am wondering about is, why is the target 1500 and not 400?

Reply:
You are obviously correct, we should be using `provideInitiative()` here. Will revamp in a few days (Monday?). We're entering quarantine and it's been hectic.

Reply:
@taltamir That's... very incorrect. `provide____` will not only maximize in the correct manner, but it will also apply a plethora of other buffs. And `autoMaximize`'s other parameters are not a target, they mirror the parameters to `maximize`.

Reply:
@soolar I didn't say maximize was better, I explained what changes jeparo's code made. Specifically, replacing provideInitiative function with a maximize